modules/faas/Makefile

```diff
@@ -1,3 +1,13 @@
+PYTHON=$(call which,python)
+
+ifeq ($(GIT_TAG),)
```

Comment:
nit picky, but this gives me the impression the `GIT_COMMIT_SHORT` takes precedence over `GIT_TAG`.

can you change this to "if GIT_TAG != None then use GIT_TAG else GIT_COMMIT_SHORT" ?